docs/en/logs/logs-field-reference.asciidoc

```diff
@@ -0,0 +1,142 @@
+[[logs-fields-reference]]
+[chapter, role="xpack"]
+= Logs fields reference
+
+This section lists the required fields the {logs-app} uses to display data.
+Some of the fields listed are https://www.elastic.co/guide/en/ecs/current/ecs-reference.html#_what_is_ecs[ECS fields].
+
+IMPORTANT: Beat modules (for example, {filebeat-ref}/filebeat-modules.html[{filebeat} modules])
+are ECS-compliant so manual field mapping is not required, and all {logs-app}
+data is automatically populated. If you cannot use {beats}, map your data to
+{ecs-ref}[ECS fields] (see {ecs-ref}/ecs-converting.html[how to map data to ECS]).
+You can also try using the experimental https://github.com/elastic/ecs-mapper[ECS Mapper] tool.
+
+`@timestamp`::
+
+Date/time when the event originated.
++
+This is the date/time extracted from the event, typically representing when the event was generated by the source.
+If the event source has no original timestamp, this value is typically populated by the first time the event was received by the pipeline.
+Required field for all events.
++
+type: date
++
+required: True
++
+ECS field: True
++
+example: `May 27, 2020 @ 15:22:27.982`
+
+
+`_doc`::
+
+This field is used to break ties between two entries with the same timestamp.
++
+required: True
++
+ECS field: False
+
+
+`container.id`::
+
+Unique container id.
++
+type: keyword
++
+required: True
++
+ECS field: True
++
+example: `data`
+
+
+`event.dataset`::
+
+Name of the dataset.
++
+If an event source publishes more than one type of log or events (e.g. access log, error log), the dataset is used to specify which one the event comes from.
++
+It’s recommended but not required to start the dataset name with the module name, followed by a dot, then the dataset name.
++
+type: keyword
++
+required: True, if you want to enable and use the *View in Context* feature.
```

Comment:
The `required` information for `event.dataset` and `host.hostname` needs to be swapped around. The former is for ML jobs, and the latter helps facilitate "View in context".

Reply:
Thanks @Kerry350